pkgs/applications/misc/soapysdr/default.nix

```diff
@@ -18,24 +22,21 @@ in stdenv.mkDerivation {
     sha256 = "14fjwnfj7jz9ixvim2gy4f52y6s7d4xggzxn2ck7g4q35d879x13";
   };
 
-  nativeBuildInputs = [ cmake pkgconfig ];
-  buildInputs = [ libusb ncurses numpy swig2 python makeWrapper ];
+  nativeBuildInputs = [ cmake makeWrapper pkgconfig ];
+  buildInputs = [ libusb ncurses numpy python swig2 ];
 
   cmakeFlags = [
     "-DCMAKE_BUILD_TYPE=Release"
     "-DUSE_PYTHON_CONFIG=ON"
   ];
 
   postFixup = lib.optionalString (lib.length extraPackages != 0) ''
-    # Join all plugins via symlinking
-    for i in ${toString extraPackages}; do
-      ${lndir}/bin/lndir -silent $i $out
+    mkdir -p $out/${modulesPath}
+    for package in ${toString extraPackages}; do
+      ${lndir}/bin/lndir -silent $package/${modulesPath} $out/${modulesPath}
```

Comment:
Now the binaries are not linked anymore. As a consequence binaries from the modules, such as `SoapySDRServer`, are not wrapped any longer.

Reply:
@markuskowa i was kindof annoyed by the collisions caused by symlinking all the packages into this one.

Reply:
wouldnt it be better to wrap the binaries in the individual packages ?

Reply:
i just noticed, that we can use SOAPY_SDR_ROOT-environment-variable when we symlink everything into the soapysdr-package

Reply:
what i find weird:
- without wrapping `SoapySDRServer` running it wont find my limesdr.
- when i export `SOAPY_SDR_ROOT` first it can find it, even without it being wrapped.
which kind of contradicts `SOAPY_SDR_ROOT` being set at compile-time. but it obviously gets set at compile-time, either defaulting to `CMAKE_INSTALL_PREFIX` or by being set explicitly with `cmakeFlags` (which i tried, but doesn't make any difference).

so yeah... even tho i would prefer a `cleaner`(tm) solution we should stick to full symlinking as it as been before.
i will clean up this PR tomorrow.